_posts/2019/2019-08-19-fandime-slusne.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Fandíme slušně!
+date: '2019-08-30T08:59:00.001+02:00'
+author: Michal Maněna
+lang: cz
+tags:
+- TDD
+- BDD
+- MDD
+- agile
+---
+
+Sparta, nebo Slávie? Bohemka! Fandíme slušně. Každý člověk má určité preference a zaujímá nějaké názory.
+Od politiky, přes ženy (muže) po fotbal. My vývojáři jsme trochu odlišní “pacienti”. 
+Preferujeme různé technologie a metodiky. A stejně jako u jiných koníčků do toho více či méně dáváme emoce. 
+V tomto článku se zamýšlíám nad profesním fundamentalismem, se kterým se stále setkávám po 15 let praxe v oboru.
+Čím dál tím víc ve mne zraje přesvědčení, že lidé v naší profesi jsou nejčastěji mýlící se pracovníky oproti ostatním profesím.
+
+# Multidisciplinární obor 
+
+Z vývoje software se stal rozsáhlý multidisciplinární obor. Když jsem v něm v roce 2004 začal pracovat jako vývojář, ještě nebylo moc rozšířené rozdělovat role programátora a kodéra. 
+Javascript bylo sprosté slovo. Všichni na něj nadávali. A již v této době jsem zaznamenával fundamentální názory některých kolegů. 
+Zejména hlásáné “jedině Java”, ostatní je … (doplň sprosté slovo). Nedivil jsem se tomu. Java má velký historicky náskok, který byl podporován řadou velkých firem i vzdělávacích institucích. 
+Na trhu existovali převážně programátoři v C či Javě.  Navíc byl u těchto jazyků nutný překlad do binárního kódu a solidní objektový návrh. 
+Poptávka po webových stránkách, resp. aplikací však často neležela v robustních aplikacích, ale často relativně primitivních stránkách s jednotkami formulářů. 
+Tyto okolnosti vedly k rozvoji vhodnějších scriptovacích jazyků s minimálními požadavky na objektový návrh, tedy zcela opačné paradigma, než bylo tou dobou u rozšířených jazyků. 
+Většina Java vývojářů pak fundamentálně odsuzovali jazyky jako PHP a toto stigma si s sebou stále neoprávněně nese, jelikož je to právě programátor, kdo je vládcem kódu. Dodržuje-li správné návyky v konkrétním jazyce, tak by měl napsat kvalitní kód. 
```

Comment:
(Většina) odsuzovali -> (Většina) odsuzovala

Reply:
Plus není úplně jasné, že si to stigma nese PHP, to spojení s vládcem kódu je moc rychlý. Dvě věty?

Reply:
Fixed